misc/scripts/download.sh

```diff
@@ -24,25 +24,34 @@
 
 # This script downloads pacscripts from the interwebs
 
-if curl --output /dev/null --silent --head --fail -- "$URL" ; then
+http_code="$(curl -o /dev/null -s -w "%{http_code}\n" -- "${URL}")"
+if [[ "${http_code}" != "200" ]]; then
+	if [[ "${http_code}" == "000" ]]; then
+		fancy_message error "Failed to download file, check your internet connection"
+		exit 1
+	fi
+	fancy_message error "Package does not exist"
+	exit 1
```

Comment:
Do this on `404` specifically, and for other non-`200` http codes print an error message with the http code.